### PR TITLE
Profile picture using assets V3

### PR DESCRIPTION
**Pull request by vytis**
Changes on UI are tiny, all functionality is in:

- [x] https://github.com/wireapp/wire-ios-sync-engine/pull/328
- [x] https://github.com/wireapp/wire-ios-data-model/pull/218

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Registration/ProfilePictureStepViewController.m [hunk 1]
```diff
@@ -295,10 +295,10 @@ - (void)setPictureImageData:(NSData *)imageData
         [[ZMUserSession sharedSession] checkNetworkAndFlashIndicatorIfNecessary];
         
         if ([self.editabledUser isKindOfClass:[ZMIncompleteRegistrationUser class]]) {
-            self.editabledUser.originalProfileImageData = imageData;
+            [[ZMUserSession sharedSession].profileUpdate updateImageWithImageData:imageData];
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Registration/ProfilePictureStepViewController.m]: It still makes sense to have a `if` `else` here?

Comment (vytis) [inline Wire-iOS/Sources/UserInterface/Registration/ProfilePictureStepViewController.m]: Yeah probably not, will update

**Comment by vytis** [comment]
@typfel updated. Or should I wrap it in `enqueueChanges` instead?

**Comment by typfel** [comment]
@vytis if it modifies a core data object then yes.

**Comment by vytis** [comment]
Updated

**Pull request merged**